CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 0.8.0 - unreleased
+
+- Update `asynchronous-codec` to `0.8` (#71)
```

Comment:
@thomaseizinger I believe you meant `0.7` 😁 

```suggestion
- Update `asynchronous-codec` to `0.7` (#71)
```

Reply:
Absolutely did!